frontend/containers/discover-map/component.tsx

```diff
@@ -148,6 +154,24 @@ export const DiscoverMap: FC<DiscoverMapProps> = ({ onSelectProjectPin }) => {
           <MapHelp />
         </div>
 
+        <div className="absolute bottom-2 left-4">
+          {displayResourceWatchCredits && (
+            <span className="text-xs text-gray-800 drop-shadow-[1px_1px_0px_rgb(198,198,198)]">
```

Comment:
The attributions are not really legible. Like the example in the [story](https://vizzuality.atlassian.net/browse/LET-737), I would suggest adding a background with low opacity. Can you check with Andreia the best way to tackle this?

![image](https://user-images.githubusercontent.com/6073968/193212945-cc2b2e89-9360-437e-b8be-fd45ab1e894a.png)